app_dart/lib/src/request_handlers/github_webhook.dart

```diff
@@ -7,7 +7,7 @@ import 'dart:convert';
 import 'dart:io';
 
 import 'package:crypto/crypto.dart';
-import 'package:github/server.dart';
+import 'package:github/github.dart';
```

Comment:
I'm surprised that so many things changed on this file and nothing changed in tests, please add tests for the use cases that changed the number of parameters and or types.

Reply:
This is expected as this PR is a pure refactor, and the tests are using public APIs while the changed methods (`_scheduleIfMergeable`, `_isIgnoredForGold`, `_checkForGoldenTriage`, `_checkForLabelsAndTests`, `_applyLabels`) are implementation details. See http://go/unit-test-practices?polyglot=dart#public-apis